### PR TITLE
ArC dev mode - include decorators in /q/arc endpoints and DEV UI

### LINKED ISSUE #23714: Decorator beans to not appear in /q/arc/beans or /q/arc/removed-beans
### Describe the bug

Decorator beans do not appear in the list of beans nor in the list of removed beans.

### Expected behavior

Decorator beans should appear in both lists as appropriate, just like Interceptor beans do.

### Actual behavior

Interceptors appear (even have their own graphical list in dev console UI), decorators do not.

### How to Reproduce?

Define a Bean with an interface and a decorator for said interface, e.g.
```
public interface EventCollector {

    List<Object> getFiredEvents();

    <T> void collect(T event);
}


@ApplicationScoped
@Unremovable
public class EventCollectorImpl implements EventCollector {

    List<Object> firedEvents = new ArrayList<>();

    @Override
    public List<Object> getFiredEvents() {
        return firedEvents;
    }

    @Override
    public <T> void collect(final T event) {
        firedEvents.add(event);
    }
}

@Decorator
@Priority(Interceptor.Priority.APPLICATION)
public abstract class EventCollectorDecorator implements EventCollector {

    @Inject
    @Delegate
    @Any
    EventCollector delegate;
}
```
The run quarkus:dev and look in /q/arc/beans, /q/arc/removed-beans, /q/dev/io.quarkus.quarkus-arc/beans, and /q/dev/io.quarkus.quarkus-arc/removed-beans for the decorator.

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

11.0.11, vendor: AdoptOpenJDK, runtime: C:\Program Files\AdoptOpenJDK\jdk-11.0.11.9-hotspot

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

2.7.1.Final

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

Apache Maven 3.8.1 (05c21c65bdfed0f71a2f2ada8b84da59348c4c5d)

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by mkouba**
- resolves #23714

### CODE REVIEW

File: extensions/arc/deployment/src/main/resources/dev-templates/decorators.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        openInIDE($(this).text());
+      });
+    });
+
+  {/script}
+
+  {#title}Decorators{/title}
+  {#body}
+  <div class="alert alert-primary alert-dismissible fade show" role="alert" data-timer="30000">
+    Decorators are sorted by the interceptor class name in ascending order. However, application decorators go first.
```
Comment (manovotn) [inline extensions/arc/deployment/src/main/resources/dev-templates/decorators.html:39]:
```suggestion
    Decorators are sorted by the decorator class name in ascending order. However, application decorators go first. 
```

**Review by manovotn — APPROVED** [review]
Tried with a quickstart and it looks good; only found one typo.

BTW I was thinking, if there is 0 decorators (or removed decorators), shouldn't we skip that section? Or perhaps instead of showing an empty table just display a note that there are none?

**Comment by mkouba** [comment]
@manovotn it should be fixed now.

**Review by manovotn — APPROVED** [review]

**Comment by mkouba** [comment]
Merging, as the timeout failure in the `integration-tests/grpc-hibernate` (JVM Tests - JDK 11) does not seem to be related.

**Pull request merged**